src/batou/environment.py

```diff
@@ -454,15 +467,23 @@ def configure(self):
                 except ConfigurationError as e:
                     # A known exception which we can report gracefully later.
                     exceptions.append(e)
-                    retry.add(root)
+                except SilentConfigurationError as e:
+                    # A silent exception which we can ignore.
+                    pass
                 except Exception as e:
                     # An unknown exception which we have to work harder
                     # to report gracefully.
                     ex_type, ex, tb = sys.exc_info()
                     exceptions.append(
-                        UnknownComponentConfigurationError(root, e, tb)
+                        UnknownComponentConfigurationError.from_context(
+                            root, e, tb
+                        )
                     )
-                    retry.add(root)
+                else:
+                    # configured this component successfully
+                    # go to next component
```

Comment:
go to next component is what the "continue" already does. intentional this is:

configured this component successfully, so we won't have to retry it later